hack/verify-generated-files-remake.sh

```diff
@@ -56,7 +56,7 @@ function newer() {
         if [[ "${F}" -nt "$2" ]]; then
             echo "${F}"
         fi
-    done
+    done | sort
```

Comment:
some other places add `LC_ALL=C` to avoid locale-specific sort differences... does that matter for these?

Reply:
those are package names which probably do not fall under locale-specific sorting. But adding LC_ALL=C certainly does not harm.

Reply:
done